administrator/components/com_admin/sql/updates/mysql/4.0.0-2021-06-08.sql

```diff
@@ -0,0 +1,23 @@
+--
```

Comment:
This and the other update SQL script should be named like `4.0.1-2021-06-08.sql` because we are in the 4.1-dev branch here and not 4.0-dev, and so these script will go into version `4.1.0` and not `4.0.0`.

Reply:
General hint on renaming or moving files: If you want to do that and also want to make changes of the file's content, do that with separate commits, i.e. one commit for rename or move, and another commit for the content changes. Otherwise GitHub will not show a moved file with content changes but a deleted and another created file.

Reply:
ok 
it's `4.1.0-2021-06-08.sql` or `4.0.1-2021-06-08-sql`

Reply:
ah typo, 4.1.0 of course. will correct previous comment.